tools/hive/kerberos/Dockerfile

```diff
@@ -1,5 +1,14 @@
 FROM cdh5-hive
 
+ENV SENTRY_VERSION 1.5.1
```

Comment:
Question. Do we ever want to use the *non* kerberized Hive? If not, we could just drop this all into a single Dockerfile.